src/js/utils/ServiceSpecUtil.js

```diff
@@ -0,0 +1,81 @@
+import PodSpec from '../structs/PodSpec';
+import ApplicationSpec from '../structs/ApplicationSpec';
+import FrameworkSpec from '../structs/FrameworkSpec';
+import Util from './Util';
+
+var ServiceSpecUtil = {
+
+  /**
+   * Creates a new PodSpec with the scaling specifications
+   * changed to fixed, and with the number of instances defined
+   * from the second argument.
+   *
+   * @param {PodSpec} spec - The spec to change
+   * @param {number} instancesCount - The new number of instances
+   * @returns {PodSpec} - Returns a new PodSpec instance
+   */
+  setPodFixedScaling(spec, instancesCount) {
+    var newSpec = Util.deepCopy(spec.get());
+    if (!newSpec.scaling) {
+      newSpec.scaling = {};
+    }
+    if (newSpec.scaling.kind !== 'fixed') {
+      newSpec.scaling = {
+        kind: 'fixed',
+        instances: 1
+      };
+    }
+
+    newSpec.scaling.instances = instancesCount;
+    return new PodSpec(newSpec);
+  },
+
+  /**
+   * Create a new ApplicationSpec with the instances property
+   * set to the instancesCount specified.
+   *
+   * @param {ApplicationSpec} spec - The spec to change
+   * @param {number} instancesCount - The new number of instances
+   * @returns {ApplicationSpec} - Returns a new ApplicationSpec instance
+   */
+  setApplicationInstances(spec, instancesCount) {
+    var newSpec = Util.deepCopy(spec.get());
+    newSpec.instances = instancesCount;
+    return new ApplicationSpec(newSpec);
+  },
+
+  /**
+   * Create a new FrameworkSpec with the instances property
+   * set to the instancesCount specified.
+   *
+   * @param {FrameworkSpec} spec - The spec to change
+   * @param {number} instancesCount - The new number of instances
+   * @returns {FrameworkSpec} - Returns a new FrameworkSpec instance
+   */
+  setFrameworkInstances(spec, instancesCount) {
```

Comment:
Sure, but since they are semantically different I wanted to separate them, just in case the way a framework scales is different. (plus we already have 2 different specs).